docs/api-guide.md

```diff
@@ -216,177 +231,191 @@ Set<String> namespaces = admin.getNamespaceNames();
 You can get the tables of a namespace as follows:
 
 ```java
-// Get the tables of the namespace "ns"
+// Get the tables of the namespace "ns".
 Set<String> tables = admin.getNamespaceTableNames("ns");
 ```
 
-### Get a table metadata
+### Get table metadata
 
-You can get a table metadata as follows:
+You can get table metadata as follows:
 
 ```java
-// Get a table metadata of "ns.tbl"
+// Get the table metadata for "ns.tbl".
 TableMetadata tableMetadata = admin.getTableMetadata("ns", "tbl");
 ```
 
-### Operations for Coordinator tables
+### Specify operations for coordinator tables
```

Comment:
I guess it would be helpful to explain what the coordinator table is here. Users would not know the coordinator table, and it could be hard for users to consider whether they should create a coordinator table or not.

Reply:
Good point🤔

I'm not exactly sure which transaction managers (other than [Consensus Commit](https://scalardb.scalar-labs.com/docs/latest/configurations/#basic-configurations)) need to have coordinator tables. Would the following be accurate? Or are there any other transaction managers we should include? (cc: @brfrn169)

> **Note**
> 
> The following is an excerpt from the "Overview of the Admin API" section in the blog post [ScalarDB Admin API & Schema Loader](https://medium.com/scalar-engineering/scalardb-admin-api-schema-loader-b0ad3809d4ea).

```suggestion
### Specify operations for coordinator tables

If you are using the Consensus Commit transaction manager, you must create coordinator tables to execute transactions. Coordinator tables are used by the [Transactional API](#transactional-api) to track the statuses of transactions.
```

Reply:
I think so, but I'm not 100% sure. I would ask @brfrn169 to check this.

Reply:
As discussed with @brfrn169, all transaction managers need Coordinator tables. I've updated this sentence to reflect that. PTAL!